src/types/runtime/gravity/Video.ts

```diff
@@ -0,0 +1,31 @@
+import { GraphQLInt, GraphQLObjectType, GraphQLString } from "graphql"
```

Comment:
This file should live in [src/schema/v2](https://github.com/artsy/metaphysics/tree/main/src/schema/v2) alongside the rest of the schema

Reply:
cool, thanks for the heads up. hoping this solves another problem I ran into. 🤞🏽